Plugins/komplugins/Komv4_plugin.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2014-2018, Els_kom org.
+// https://github.com/Elskom/
+// All rights reserved.
+// license: MIT, see LICENSE for more details.
+
+namespace komv4_plugin
+{
+    public class Komv4_plugin : Els_kom_Core.interfaces.IKomPlugin
+    {
+        public string KOMHeaderString => "KOG GC TEAM MASSFILE V.0.4.";
+        public int SupportedKOMVersion => 4;
+
+        public void Pack(string in_path, string out_path, string KOMFileName)
+        {
+            // not implemented yet due to lack of packing information on v4 koms.
+            throw new System.NotImplementedException();
```

Comment:
This is still not implemented. Need to figure out implementing this packing method.